v0/pipeline/steps/idp/idpcookiecutter-info.yaml

```diff
@@ -0,0 +1,42 @@
+title: IDPCookieCutterPlanInfo
+allOf:
+  - $ref: ../../common/step-spec-type.yaml
+  - type: object
+    required:
+    - isPublicTemplate
+    - cookieCutterVariables
+    properties:
+      isPublicTemplate:
```

Comment:
Just confirming if we really need this? Or can we infer based on which one of publicTemplateUrl or pathForTemplate is set?

Reply:
I added it based on the discussion and thought if needed by UI, once UI integration is done if we don't need it, we will remove it. Correct from the backend we can handle it without this variable as well.